tests/Fallback/AssetFallbackTest.php

```
@@ -77,124 +39,152 @@ public static function getSaveData(): array
+        file_put_contents($path, '{}');
 
-        \file_put_contents($path, '{}');
-        $this->assertFileExists($path);
+        $this->io->expects(self::once())->method('write');
 
-        MockerState::addCondition('Foxy\\Fallback', 'file_get_contents', ['package.json', false, null, 0, null], false);
+        $this->fs
+            ->expects(self::once())
+            ->method('remove')
+            ->with('package.json')
+            ->willThrowException(new \RuntimeException('Remove failed.'));
```

PHPMD (2.15.0)

Missing class import via use statement (line '79', column '38'). (undefined)

(MissingImport)